### PR TITLE
Replace xme_ca_udm_dbg with xca_udm_dbg --aiearch aie-ml in the unit_tests/aievec_tests

### PR DESCRIPTION
**Pull request by linay-xsj**

### CODE REVIEW

File: test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=16 test-fastest-varying=0 vectorize-reductions=true" --convert-vector-to-aievec="aie-target=aieml" -lower-affine | aie-translate -aieml=true --aievec-to-cpp -o dut.cc
 // RUN: xchesscc_wrapper aie2 -f -g +s +w work +o work -I%S -I. %S/testbench.cc dut.cc
 // RUN: mkdir -p data
-// RUN: xme_ca_udm_dbg -qf -T -P %aietools/data/aie_ml/lib/ -t "%S/../profiling.tcl ./work/a.out" >& xme_ca_udm_dbg.stdout
-// RUN: FileCheck --input-file=./xme_ca_udm_dbg.stdout %s
+// RUN: xca_udm_dbg --aiearch aie-ml -qf -T -P %aietools/data/aie_ml/lib/ -t "%S/../profiling.tcl ./work/a.out" >& xca_udm_dbg --aiearch aie-ml.stdout
+// RUN: FileCheck --input-file=./xca_udm_dbg --aiearch aie-ml.stdout %s
```
Comment (stephenneuendorffer) [inline test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir]: Oh cool!  I didn't know that worked.  Is this the preferred way of running the aie2 simulator?

Comment (linay-xsj) [reply test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir]: xme_ca_udm_dbg is obsolete today. xca_udm_dbg --aiearch aie-ml is a preferred way of running the aie2 simulator now.

Comment (fifield) [reply test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir]: With which version of Vitis does `xca_udm_dbg --aiearch aie-ml` start? I'm seeing errors related to this in my builds.

Comment (linay-xsj) [reply test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir]: daily_latest

Comment (david-vc) [reply test/unit_tests/aievec_tests/bf16_add_reduce/bf16_add_reduce.mlir]: for 2022.2 use xme_ca_udm_dbg;
for 2023.1 and later use xca_udb_dbg --aiearch aie-ml

**Pull request merged**